org.eclipse.xtext.util/src/org/eclipse/xtext/util/UriExtensions.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2017, 2019 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.xtext.util;
+
+import java.net.URISyntaxException;
+
+import org.eclipse.emf.common.util.URI;
+
+/**
+ * @author Christian Dietrich - Initial contribution and API
+ * @since 2.14
+ */
+public class UriExtensions {
+	/**
+	 * returns a URI with empty authority, if absent and has file scheme.
```

Comment:
This comment doesn't make any sense, and doesn't start with a capital letter.

`if absent and has file scheme` if what's absent? If what has a file scheme?
Same comment for the other method comments in this class

Reply:
I have added a test case and made the comments hopefully a bit more explicit.